kroxylicious-operator/DEV_GUIDE.md

```diff
@@ -52,7 +52,7 @@ should produce a `target/kroxylicious-operator-0.10.0-SNAPSHOT-bin` directory.
 Build the operator image. For development purposes you can use the minikube registry directly using `minikube image build`, which will be faster than alternatives like pushing to quay.io from your host only for kube to pull the same image right back when you deploy the operator.
```

Comment:
Maybe we should add instructions for how to do the quay build/push with `build-image.sh` to demonstrate how it's done.